auth_password_settings/__openerp__.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Dhaval Patel
+#    Copyright (C) 2011 - TODAY Denero Team. (<http://www.deneroteam.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'User Password Settings',
+    'version': '1.0',
+    'category': 'base',
+    'sequence': 3,
+    'summary': 'User Password Settings',
+    'author': 'Dhaval Patel<dhaval@deneroteam.com>',
```

Comment:
please add 'Odoo Community Association (OCA)' to the list of authors so that the modules shows up as an OCA addon on apps.odoo.com.